web/modules/custom/joinup_notification/config/install/message.template.og_membership_approve_with_subscription.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies: {  }
+template: og_membership_decision
+label: 'Group membership approval/rejection'
+description: 'Group privileged user decides on a membership request.'
+text:
+  -
+    value: "<p>Joinup: Your request to join the @entity:bundle @entity:title was approved</p>"
+    format: full_html
+  -
+    value: "<p>@actor:full_name has approved your request to join the \"@entity:title\" @entity:bundle.</p><p>You will receive weekly notifications for newly created content on this collection.<br /> To manage your notifications go to \"<a href=\"@user:my_subscriptions\">My subscriptions</a>\" in the user menu.</p><p>If you think this action is not clear or not due, please contact Joinup Support at <a href=\"@site:contact_url\">@site:contact_url</a>.</p>"
```

Comment:
Here also "...to join **and subscribe to** the..."

Reply:
Sorry, it seems I removed almost all instances of this old renaming but forgot this. The `og_membership_approve_with_subscription` is more than 32 characters and cannot be used as a message type. I had to shorten it to the `og_membership_subscribed_approve` above (which is exactly 32 characters). Removed this one.